learning/katas/go/Core Transforms/Map/ParDo OneToMany/pkg/task/task.go

```diff
@@ -21,10 +21,10 @@ import (
 )
 
 func ApplyTransform(s beam.Scope, input beam.PCollection) beam.PCollection {
-	return beam.ParDo(s, processFn, input)
+	return beam.ParDo(s, tokenizeFn, input)
 }
 
-func processFn(input string, emit func(out string)) {
+func tokenizeFn(input string, emit func(out string)) {
```

Comment:
The answer placeholder(s) seems weird here. Maybe it requires a fix?
Or alternatively, we can use a similar answer placeholder as in the "ParDo" task, i.e. having just empty "fund ApplyTransforms"?
![image](https://user-images.githubusercontent.com/5459430/80789567-e88fb280-8bbe-11ea-9185-b87c867f1bee.png)

Reply:
Similar to the previous placeholder issue.  This is what appears when I tested the course in both IntelliJ and GoLand:

<img width="717" alt="Screen Shot 2020-05-03 at 11 18 41 AM" src="https://user-images.githubusercontent.com/762456/80922546-bacb7a80-8d32-11ea-9486-524d57333235.png">

Reply:
This looks good now

Reply:
This looks good now